components/kyma-environment-broker/cmd/broker/main.go

```diff
@@ -158,6 +160,10 @@ func main() {
 
 	logs := logrus.New()
 	logs.SetFormatter(&logrus.JSONFormatter{})
+	if cfg.LogLevel != "" {
```

Comment:
as I understand it is done by the way ;)

Reply:
yes, I realized it is missing